Procfile.dev

```diff
@@ -1,4 +1,4 @@
 server: bin/rails server -p 3020
 assets: bin/webpack-dev-server
 anycable: RAILS_ENV=development bundle exec anycable --server-command "anycable-go --host=localhost --port 3334"
-js-tests: yarn test-watch
+#js-tests: yarn test-watch
```

Comment:
Was this intentional?

Reply:
Nope. It was in my procfile for some reason. So. Many. Branches.